### ISSUE TITLE
have you tested it ?

### ISSUE THREAD

**Issue by china56321** [syedmutti/Semi-AutomatedAnnotation#1]
can you write the procedure in detail ? thank you !

**Comment by syedmutti** [comment]
This was supposed to be used for a private project. It creates annotations based on trained network.
Can you let me know what exactly you want to achieve ? 
Do you want to annotate objects (e.g from MS COCO dataset) on your custom data ?

**Comment by china56321** [comment]
Yes,I want to annotate objects on my own datasets automatically ,but most of them are consistent with MS COCO datasets (person, car ,bus, truck, bicycle,motorbike,Six Categories),
The purpose of mine is to use a  pre-trained models to detect images ,then save the label ,finally fine-tune the label  with LabelIMG manually (that's so-called Semi-Automated-Annotation),Could it be achieved ? Could you provide me your scripts and implement steps in detail? thanks very much !






At 2019-11-19 17:06:07, "syedmutti" <notifications@github.com> wrote:


This was supposed to be used for a private project. It creates annotations based on trained network.
Can you let me know what exactly you want to achieve ?
Do you want to annotate objects (e.g from MS COCO dataset) on your custom data ?

**Comment by syedmutti** [comment]
So, that is what it exactly does.

Step 1:  Get a trained model from Tensorflow Object Detection API and try to infer objects using Quickstart Notebook. 

API Installation Instructions : https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/installation.md

Once you have installed , use Quickstart Notebook to load a model trained on MSCOCO from Tensorflow Model Zoo : https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md

Step 2 : Once you can use Object Detection API to infer on images, first replace the given "visualization_utils.py" with the file at folder "../resarch/object detection/utils/visualization_utils.py"

Step 3 : Replace the last 2 Cells from my notebook with "Quickstart Notebook".

It will create xml files that can be read directly with annotation tools like "Labelimg" etc.

Note :  Ofcourse, You also need to chage the file directory for input images, model to use and label maps in "Quickstart Notebook" - Better read what Quickstart notebook is doing.
Also, if you have not installed Tensorflow Object Detection API, it might need a little time to install and get comfortable with.

Good Luck
